src/main/kotlin/lotto/Purchase.kt

```diff
@@ -0,0 +1,26 @@
+package lotto
+
+data class Purchase(val amount: Int) {
```

Comment:
Why did you use a data class? Is it really necessary? 🤔

Reply:
Sure! Please look deeper into the difference between data class and value class.

In this case, value class seems more appropriate, as it’s well-suited for wrapping a single primitive property to represent a domain model. It also comes with the bonus of no runtime wrapping overhead, which means better performance.

Give it a try! 🙂